tests/e2e/lib/pages/wp-admin/plugins.js

```diff
@@ -33,23 +33,22 @@ export default class PluginsPage extends Page {
 	}
 
 	async getJetpackVersion() {
-		const versionText = 'tr.active[data-plugin="jetpack/jetpack.php"] .plugin-version-author-uri';
+		const versionText = 'tr.active[data-plugin$="/jetpack.php"] .plugin-version-author-uri';
```

Comment:
Any reason why do we change the locator here? 

It seems that with this change it would look for `jetpack-dev/jetpack.php` as well, which is not desired.

Reply:
You are right. I actually wanted to make it look for `jetpack-dev`, in an attempt to fix the failing test in a local env, not knowing at the time that update test should not work with dev plugin. I will change it back.